app/services/l_s_extract.rb

```diff
@@ -124,8 +191,9 @@ def generate_report(job, sales)
   end
 
   def get_report_line(job, sale, products, customers)
-    last_name = sale["Customer"]["lastName"].gsub(/\*\d+\*$/, "").strip.tr("*", "")
-    tax_total = (sale["calcTax1"].to_f.round(2) + sale["calcTax2"].to_f.round(2)).round(2)
+    last_name = sale["Customer"]["lastName"].gsub(/\*\d+\*$/, "").strip.tr('*','')
```

Comment:
Our code style guide prefers double quotes. You can run `bundle exec standardrb --fix` to fix any style issues.